services/thumbnails/README.md

```diff
@@ -89,3 +89,29 @@ To have more control over memory (and CPU) consumption the maximum number of con
 ## Thumbnails and SecureView
 
 If a resource is shared using SecureView, the share reciever will get a 403 (forbidden) response when requesting a thumbnail. The requesting client needs to decide what to show and usually a placeholder thumbnail is used.
+
+## Using libvips for thumbnail generation
+
+To improve performance (and to support a wider range of images formats) the thumbnails service is able to utilize libvips (https://www.libvips.org/) for thumbnail generation. Support for libvips needs to be
+enabled at buildtime and has a couple of implications:
+
+*  With libvips support enabled it is not possible to create a statically linked ocis binary
+*  So the libvips shared libraries need to be available at runtime in the same release that was used to build the ocis binary
+*  When using the ocis docker images the libvips shared libraries are included in the image
+
+Support of libvips is disabled by default. To enable it make sure libvips and its buildtime dependencies are install in your build environment. Then just set
+the `ENABLE_VIPS` variable on the `make` command:
```

Comment:
```suggestion
Support of libvips is disabled by default. To enable it, make sure libvips and its buildtime dependencies are installed in your build environment.
Then you just need to set the `ENABLE_VIPS` variable on the `make` command:
```
Does one know how `buildtime dependencies are installed` works?
Should we add some text/links?

Reply:
I tend to think that somebody trying to build ocis himself/herself should know how to install build dependencies.